server/market/market.go

```diff
@@ -38,6 +39,7 @@ const (
 // Swapper coordinates atomic swaps for one or more matchsets.
 type Swapper interface {
 	Negotiate(matchSets []*order.MatchSet)
+	TxMonitored(user account.AccountID, asset uint32, txid string) bool
```

Comment:
It seems prudent to specify the asset given that an asset may be structured in a way that makes cloning a txid from another asset possible.

Reply:
Yep. I actually have a comment to this effect in the review I'm working on for #85.